extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeMusicSearchExtractor.java

```diff
@@ -88,6 +90,7 @@ public void onFetchPage(@Nonnull final Downloader downloader) throws IOException
                     .end()
                     .object("activePlayers").end()
                     .object("user")
+                        // TO DO: provide a way to enable restricted mode with:
```

Comment:
Here is also a Todo about restricted mode

Reply:
Yes, but not in this PR.